src/item.cpp

```diff
@@ -825,7 +825,41 @@ std::string Item::getDescription(const ItemType& it, int32_t lookDistance,
 				tmpSubType = item->getSubType();
 			}
 
-			s << ". " << (it.stackable && tmpSubType > 1 ? "They" : "It") << " can only be used with";
+			s << ". " << (it.stackable && tmpSubType > 1 ? "They" : "It") << " can only be used by ";
+
+			VocSpellMap vocMap = g_spells->getRuneSpell(it.id)->getVocMap();
```

Comment:
1) Doesn't that means someone has not configured a rune properly? By setting it as a rune but not assigning any callback to it. Maybe it can lead to other problems too.
2) No, my bad, will fix.